src/wp-includes/global-styles-and-settings.php

```diff
@@ -225,6 +225,39 @@ function wp_get_global_stylesheet( $types = array() ) {
 	return $stylesheet;
 }
 
+/**
+ * Gets the global styles custom css from theme.json.
+ *
+ * @since 6.2.0 Added custom css support to theme.json.
```

Comment:
As this is a brand new function being introduced into Core, the explanation after the `@since` is not needed.
```suggestion
 * @since 6.2.0
```